.dockerignore

```diff
@@ -0,0 +1,22 @@
+#editor
+.prettierrc
+.vscode
+
+#test
+coverage
```

Comment:
why we need  `coverage` here?  It doesn't exist in repo. 
I can see only **one** big directory in ignore file - node_modules.

Reply:
`npm run test:cov` will generate the coverage report

Reply:
yeah, i found it too, thanks